.github/workflows/go.yml

```diff
@@ -92,3 +92,48 @@ jobs:
         with:
           version: latest
           args: --timeout=5m
+
+  codeql:
+    name: CodeQL
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'go' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
```

Comment:
the more i read this, the more i am not sure about this comments

Reply:
Fixed.